GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+# Eiffel Community Governance
+
+Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability.
+Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
+
+Governance of the Eiffel Community is structured according to this document.
+This structure in intended to provide a healthy, sustainable and attractive community for Eiffel development and collaboration.
+
+The Eiffel community proactively adopts and follows best practices employed by successful open source communities while maintaining a governance agreeable to it's members.
+
+# Governance
+
+Eiffel is a community of projects working on building CI/CD Pipelines using Eiffel Protocol.
+
+The projects that constitute Eiffel Community are self governing, meaning that project related work is done and decisions are taken within the projects.
+
+Governance of the Eiffel Community is the responsibility of Eiffel Technical Committee.
+
+## Projects
+
+The projects that take part in Eiffel Community develop solutions in order to contribute to Eiffel vision of technology agnostic machine-to-machine communication in CI/CD pipelines.
+The work done within the projects can focus on specific areas such as security, visualization, and ease of use or contribute to and drive the high level architecture or the protocol itself which impacts the entire community.
+
+The projects that take part in Eiffel Community currently have two roles relevant to their governance.
+
+### Contributors
+
+A contributor is anyone who made contributions to the projects within Eiffel Community.
```

Comment:
Instead of "to the projects" we could say "to any of the projects", making it more clear that there is no obligation to contribute to all projects, it could be just one.

Reply:
will fix.